src/universal/redux/getAuthedUser.js

```diff
@@ -4,6 +4,7 @@ query {
     email,
     id,
     picture,
+    picturePutUrl,
```

Comment:
I am open to sticking `picturePutUrl` elsewhere. The two options as I saw it was: 1) adding a graphql schema who's job it was to provide CDN put urls, or 2) extend an existing type

This seemed like a reasonable starting place...